src/main/java/com/tabnine/inline/InlineCompletionHandler.java

```diff
@@ -69,14 +69,15 @@ public void retrieveAndShowCompletion(
       }
     } else {
       ObjectUtils.doIfNotNull(lastPreviewTask, task -> task.cancel(false));
-      long taskDelayTime = CompletionTracker.calcDebounceTime(editor);
+
+      List<TabNineCompletion> completions =
+          retrieveInlineCompletion(editor, offset, tabSize, completionAdjustment);
```

Comment:
where is this executed? on the UI thread?

Reply:
no, in "invokeLater" thread

Reply:
I thought invokeLater is not executed on the UI thread. I changed it.